src/components/Footer/Footer.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+
+const Footer = () => {
+  const location = useLocation();
+
+  let isCheckPath = location.pathname === '/';
+
+  return (
+    <FooterContainer>
+      <Inner
+        style={{
+          maxWidth: isCheckPath ? 'none' : '1120px',
+          padding: isCheckPath ? '0 80px' : '0',
+        }}
```

Comment:
스타일 컴포넌트의 특징은 컴포넌트에 props로 데이터를 내려주는 것 처럼
이 부분에 props로 데이터를 지정해주면
스타일 부분에서 사용할 수 있다는 점입니다.

```suggestion
      <Inner path={isCheckPath}
```
와 같이 넘겨주면 아래에서 props를 이용해서 스타일에 삼항연산자를 사용할 수 있습니다.